aea/identity/base.py

```diff
@@ -70,6 +70,12 @@ def __init__(
         if addresses is None:
             addresses = {default_address_key: address}
         self._addresses = addresses
+        self._default_address_key = default_address_key
```

Comment:
For consistency, could be relabelled to `default_ledger_id`

Reply:
ok